build.gradle.kts

```diff
@@ -15,16 +15,16 @@ import org.json.JSONObject
 // Although this allows different clients to use different artifacts, since we only have one client
 // this is now the most important use case for this implementation. Instead, this implementation
 // aims to make it easier to test publisher changes without having to override the artifacts.
-val publisherVersion = "v1"
+val publisherVersion = "v2"
 
 plugins {
     id("com.android.library") apply false
     id("com.automattic.android.publish-to-s3") apply false
 }
 
-val defaultCompileSdkVersion = 30
+val defaultCompileSdkVersion = 33
 val defaultMinSdkVersion = 21
```

Comment:
👋 @fluiddot !

FYI: I am noticing that the `defaultMinSdkVersion` is `21` as well, maybe we should consider updating that to `24` as some point in the future too (if `21` is not required somehow), just like we did across all other libs, included GB and Aztec. 🤔

Reply:
Sure, I can update it to match the same value we have in Aztec and GB. I also noticed we use the same value in the forked repositories ([example](https://github.com/wordpress-mobile/react-native-gesture-handler/blob/022b08e4bbbcadd9f84cdeda7087b38f696ad4f2/android/build.gradle#L59)). I can update it also there. Thanks for the call out 🙇 !

Reply:
I applied this suggestion in https://github.com/wordpress-mobile/react-native-libraries-publisher/pull/17/commits/be2d692690507b9f273a2824aa482476cac54eeb.

Reply:
Awesome, thank you so much for dealing with that @fluiddot , you rock! 🙇 ❤️ 🚀